src/main/resources/tinylog.properties

```diff
@@ -11,3 +11,12 @@ exception = strip: jdk.internal
 level@org.jabref.gui.maintable.PersistenceVisualStateTable = debug
 
 level@org.jabref.http.server.Server = debug
+
+level@org.jabref.logic.ai.AiIngestor = trace
+
+# TODO: Set up properly.
+langchain4j.open-ai.chat-model.log-requests = true
+langchain4j.open-ai.chat-model.log-responses = true
+logging.level.dev.langchain4j = DEBUG
```

Comment:
You forgot the prefix `level@` at each line. I know, its a strange syntax, but one needs to do like that ^^.

You need to state classes here, not the configuration property.

Reply:
We've talked about logging on Monday's devcall.

Log story short, it turns out it's enough to run JabRef with `--debug` argument and `langchain4j` will catch that and log responses.

I'll just clear that

Reply:
Nevertheless, you can fine-tune the output even with `--debug` presence. -- `--debug` is the same as changing line 1 of the `tinylog.properties`.